### PR TITLE
 Add sig-usability subproject 

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/org/issues/1047

/assign @tashimi @vllry @Rajakavitha1 @hpandeycodeit

### CODE REVIEW

File: sig-usability/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - sig-usability-leads
+approvers:
+  - sig-usability-leads
+labels:
+  - sig/usability
```
Comment (nikhita) [inline sig-usability/OWNERS]: Created https://github.com/kubernetes/test-infra/pull/13676 to add the label

**Comment by castrojo** [comment]
/lgtm 

Looking good so far, I'm attending the first meeting so we can double check the new SIG has everything they need.

**Comment by spiffxp** [comment]
/approve
per @vllry's /approve

**Pull request merged**